src/components/CampaignContactsForm.jsx

```diff
@@ -209,8 +209,10 @@ export default class CampaignContactsForm extends React.Component {
         Your upload file should be in CSV format with column headings in
         the first row. You must include <span className={css(styles.csvHeader)}>firstName</span>,
         <span className={css(styles.csvHeader)}>lastName</span>, and
-        <span className={css(styles.csvHeader)}>cell</span> columns. If you include a <span className={css(styles.csvHeader)}>zip</span> column,
+        <span className={css(styles.csvHeader)}>cell</span> columns.
+        If you include a <span className={css(styles.csvHeader)}>zip</span> column,
         we'll use the zip to guess the contact's timezone for enforcing texting hours.
+        An optional column to map the contact to a CRM is <span className={css(styles.csvHeader)}>external_id</span>
```

Comment:
to the csv (and in the future as a column in the DB sql), yes, but then we can update/sync/send actions based on it